gedcom/__init__.py

```diff
@@ -34,8 +34,14 @@
 __all__ = [
     # Subpackages
     "element",
+    "subparsers",
```

Comment:
Found something fairly interesting: https://softwareengineering.stackexchange.com/a/75929

I think we can just rename `element` to `elements` since it contains homogeneous contents.